trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/model/views/TableColumnHeader.java

```diff
@@ -36,11 +37,16 @@ public class TableColumnHeader {
      * @param dataModel
      *            The tree model that serves as base for this column
      */
+    @SuppressWarnings("restriction")
     public TableColumnHeader(ITmfTreeDataModel dataModel) {
         fId = dataModel.getId();
         List<@NonNull String> labels = dataModel.getLabels();
         fName = dataModel.getLabels().get(0);
-        fDescription = labels.size() >= 2 ? dataModel.getLabels().get(1) : null;
+        if (dataModel instanceof TmfEventTableColumnDataModel) {
```

Comment:
It doesn't seem right that we have to do that.
I believe that TmfEventTableDataProvider should be updated to use the TmfTreeModel.Builder and set column descriptors that include the tooltips.